bin/users

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+$stdout.sync = true
+
+$LOAD_PATH << 'lib'
+
+require 'bundler/setup'
+require 'travis/api/v3/models/user'
+
+option = ARGV[1]
+
+if option == '--active'
```

Comment:
There's a nice short way of doing assignment/if (feel free to ignore as it's not important):

```ruby
users = if option == '--active'
  User.where # etc.
else
  User.where # etc.
end
```